polaris/polaris/tests/processes/test_create_stellar_deposit.py

```diff
@@ -169,6 +169,11 @@ def test_deposit_stellar_no_trustline(acc1_usd_deposit_transaction_factory):
     `create_stellar_deposit` sets the transaction with the provided `transaction_id` to
     status `pending_trust` if the provided transaction's Stellar account has no trustline
     for its asset. (We assume the asset's issuer is the server Stellar account.)
+
+    This is the flow when a wallet/client submits a deposit without the sending
+    "claimable_balance_supported=True" in their POST request body.
+    Such that the deposit transaction is blocked
+    until the Wallet user creates a trustline to that asset
```

Comment:
Lets assert that `claimable_balance_supported` is `False` too

Reply:
Done and done.